osu.Game/Overlays/BeatmapSet/AuthorInfo.cs

```diff
@@ -50,19 +51,19 @@ private void updateDisplay()
             fields.Children = new Drawable[]
             {
                 new Field("mapped by", BeatmapSet.Metadata.Author.Username, OsuFont.GetFont(weight: FontWeight.Regular, italics: true)),
-                new Field("submitted on", online.Submitted.ToString(@"MMMM d, yyyy"), OsuFont.GetFont(weight: FontWeight.Bold))
+                new Field("submitted", online.Submitted, OsuFont.GetFont(weight: FontWeight.Bold))
                 {
                     Margin = new MarginPadding { Top = 5 },
                 },
             };
 
             if (online.Ranked.HasValue)
             {
-                fields.Add(new Field("ranked on", online.Ranked.Value.ToString(@"MMMM d, yyyy"), OsuFont.GetFont(weight: FontWeight.Bold)));
+                fields.Add(new Field("ranked", online.Ranked.Value, OsuFont.GetFont(weight: FontWeight.Bold)));
```

Comment:
This is not consistent with web - dropping the `ToString()` means that the date will get automatically humanised. See screenshots in the original issue.

Reply:
Wouldn't it make more sense to instead change web to the humanised form to be consistent with all other occurrences of time stamps?
![Capture](https://user-images.githubusercontent.com/32681550/74111396-94af9780-4b94-11ea-8f15-bec9691d4c87.PNG)
![Capture2](https://user-images.githubusercontent.com/32681550/74111400-95e0c480-4b94-11ea-89e7-0eba386c1cf2.PNG)
![Capture3](https://user-images.githubusercontent.com/32681550/74111401-95e0c480-4b94-11ea-8064-4f4ae5831253.PNG)
![Capture4](https://user-images.githubusercontent.com/32681550/74111403-95e0c480-4b94-11ea-98c6-d48954fd6246.PNG)

Reply:
I imagine this might have been an intentional choice. Note that humanised dates are used in lists of things, where it's more readable to have a human-readable offset rather than a precise date. In the case of beatmap metadata, however, it is more useful to see the full date outright without having to hover over it or trying to do mental math. I am however not sure.

cc @arflyte - could you confirm/deny if this was intentional?

Reply:
It should be matching, so short date format is best here.

Reply:
actually let's go with relative for now. it feels better.